sources/packages/backend/libs/services/src/queue/queue.service.ts

```diff
@@ -26,14 +26,23 @@ export class QueueService {
       select: {
         queueName: true,
         queueConfiguration: true as unknown,
-      },
-      where: {
         isActive: true,
       },
     });
     return this.queueConfiguration;
   }
 
+  /**
+   * Get all inactive queue names.
+   * @returns queue names.
+   */
+  async getInactiveQueueConfigurations(): Promise<QueueNames[]> {
```

Comment:
Should the function name be `getInactiveQueueNames()` ?

Reply:
Yes, it should.